projects/plugins/jetpack/modules/masterbar/admin-menu/load.php

```diff
@@ -35,6 +35,21 @@ function should_customize_nav( $admin_menu_class ) {
 		return false;
 	}
 
+	// WPCOM-specific exemptions.
```

Comment:
We currently maintain the Nav Unification activation logic for Simple sites in the `wpcom` codebase (in the `show_unified_nav` function which lives in `wp-content/mu-plugins/masterbar.php`), so I guess it makes sense to also add these new checks there?

Reply:
I did some testing (the patch is in D75746-code). It looks like it's another way to accomplish the same thing, either this PR or D75746-code is needed, but no extra benefit from both. Maybe D75746-code is better because it keeps the wpcom specific exemptions in wpcom and out of jetpack?  I'd be fine w/ closing this PR and going for D75746-code.

![2022-02-25_15-07](https://user-images.githubusercontent.com/937354/155804189-3c8e2b99-c30f-462a-bfa3-39d70c909fa2.png)
![2022-02-25_15-08](https://user-images.githubusercontent.com/937354/155804203-a4fa6461-bcf5-42fe-92f2-1849b92f06fe.png)
![2022-02-25_15-09](https://user-images.githubusercontent.com/937354/155804215-cb0f9ea1-3b2f-4444-96fa-02c67a545827.png)
![2022-02-25_15-10](https://user-images.githubusercontent.com/937354/155804222-d746a1b6-7a8e-42cf-9d51-6a8539c415a4.png)

Reply:
I think keeping wpcom's special case on wpcom in D75746-code makes sense. With https://github.com/WordPress/gutenberg/pull/38810 about to be merged you could even keep to just the `*stuff.php` file checks.

Reply:
> I'd be fine w/ closing this PR and going for D75746-code.

Let's do that then 🙂